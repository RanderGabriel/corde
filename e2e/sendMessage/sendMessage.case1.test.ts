import { runCLI } from "../cliRunner";

import Utils from "../testUtils";

it("should return the sent message", async () => {
  const command = Utils.buildCommandWithConfigPath("sendMessage", "bot_case1.test.ts");
  const result = await cli.exec(command);
  expect(result.statusCode).toEqual(0);
  expect(result.stdout).toMatchSnapshot();
});
