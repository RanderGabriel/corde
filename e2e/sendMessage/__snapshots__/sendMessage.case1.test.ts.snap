// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the sent message 1`] = `""`;
